env/webpack.prod.config.js

```diff
@@ -12,5 +12,8 @@ module.exports = webpackMerge(commonConfig, {
                 warnings: false
             }
         })
-    ]
+    ],
+    module: {
+        preLoaders: []
```

Comment:
We remove elslint check during build.  That save alot of time in raspberry during build